install/linux/linux-postinstall.md

```diff
@@ -39,14 +39,20 @@ To create the `docker` group and add your user:
     ```bash
     $ sudo usermod -aG docker $USER
     ```
-
-3.  Log out and log back in so that your group membership is re-evaluated.
+3. Restart the docker service.
+   Since the changes in ownership of the Unix socket are only done when the `docker` daemon starts, we need to restart the service.
```

Comment:
Can you confirm this @thaJeztah?

Reply:
Hm, don't think that should be needed; these instructions are for adding the current user to the docker group; no changes are made to the permissions on the socket itself?